StripeConnect/StripeConnectTests/Internal/Webview/ConnectComponentWebViewTests.swift

```diff
@@ -13,6 +13,9 @@ import WebKit
 import SafariServices
 
 class ConnectComponentWebViewTests: XCTestCase {
+
+    typealias FontSource = EmbeddedComponentManager.CustomFontSource
```

Comment:
It looks like this typealias is unused. Did you mean to refer to it on line 122?